rstar/src/algorithm/bulk_load/bulk_load_sequential.rs

```diff
@@ -4,6 +4,11 @@ use crate::object::RTreeObject;
 use crate::params::RTreeParams;
 use crate::point::Point;
 
+use alloc::{vec, vec::Vec};
+
+#[cfg(not(test))]
+use num_traits::Float;
```

Comment:
I think this is outdated now - I think every occurrence of `#[cfg(not(test))]`  (there should be three of them) can be omitted.

Reply:
If I remove it and run `cargo test` from the workspace root, I get

```console
warning: unused import: `num_traits::Float`
 --> rstar/src/algorithm/bulk_load/bulk_load_sequential.rs:8:5
  |
8 | use num_traits::Float;
  |     ^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

warning: unused import: `num_traits::Float`
 --> rstar/src/algorithm/bulk_load/cluster_group_iterator.rs:4:5
  |
4 | use num_traits::Float;
  |     ^^^^^^^^^^^^^^^^^

warning: unused import: `num_traits::Float`
  --> rstar/src/algorithm/removal.rs:10:5
   |
10 | use num_traits::Float;
   |     ^^^^^^^^^^^^^^^^^

warning: `rstar` (lib) generated 3 warnings
```

Are you sure?

Reply:
I was a little unclear there - the _import_ is still required, the `cfg(not(test))` doesn't seem to make any difference any more. I'm getting warnings with and without the `cfg(not...)` block when running `cargo test`.

Unless you have a different idea I would suggest to use this instead:

```rust
#[allow(unused_imports)] // Import is required when building without std
use num_traits::Float;
```

Reply:
Ah, you where probably running `cargo test` from the workspace root? When you run `cargo test` from the workspace root, I get the warning. When I run `cargo test` from the `rstar` folder, I do not get the warning. Either way, yes, I'll go for your fix.

Reply:
Done